examples/datadog-express-apm/host-agent/utils/response.js

```diff
@@ -30,7 +30,7 @@ class ResponseHandler {
     agent.init().dogstatsd.increment(`Commercetools_Client_Response`, 1, {
       env: 'dev',
       status_code: statusCode,
-      http_method: req.method,
```

Comment:
Why was this remove?

Reply:
because it was giving me the error on the `req` variable that is not passed

Reply:
Oh, okay. I guess we should find a way to pass the request method down to this function then.